_pages/crossbrowser-testing.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cross-browser Testing
```

Comment:
For the purposes of file naming, should the dash between "cross-browser" be reflected in the file name (`cross-browser-testing.md`)?

Reply:
Eh, probably doesn't matter too much, I haven't seen a strict practice on this in any of our guides. But based on the organizational issue you flagged above, I think this will just end up being "browser-testing.md"